app/assets/v2/js/grants/index.js

```diff
@@ -320,8 +320,13 @@ if (document.getElementById('grants-showcase')) {
         if (!append_mode) {
           vm.grants = [];
         }
+
+        const previouslyLoadedIds = vm.grants.map(grant => grant.id);
```

Comment:
Consider: moving this to the vue model to avoid creating the mapping every time we call `fetchGrants()`